### DISCUSSION TITLE
【分享】开源项目推荐

### DISCUSSION THREAD

**Discussion by 6078HITHEDAY** [share121/inter-knot#2408] [其他]
---
title: 开源项目推荐
date: 2024-04-06 10:22:31
tags: 推荐
categories: 一些项目
---
### 一些好的项目推荐
- 除特别标注，其他一般全平台,github下载寻找releases（发行）
## Miru （视频，漫画，小说）
![Miru](https://github.com/miru-project/miru-app/raw/dev/assets/icon/logo.png "miru app")
免费和开源的多功能应用程序，支持视频，漫画，小说扩展源Android，Windows，Web。
![Miru Android](https://github.com/miru-project/miru-app/raw/dev/assets/screenshot/screenshot.webp "Android")
![Miru Windows](https://github.com/miru-project/miru-app/raw/dev/assets/screenshot/screenshot.webp "Windows")
# 特征
* 支持windows和android
* 友好的扩展写入支持，调试日志
* 扩展使用JavaScript语言，开发简单
* 支持自定义扩展存储库
* 官方扩展库提供了视频源，无需编写任何扩展即可使用
* 支持在线观看视频、漫画、小说多个来源，实现多平台统一
* 统一系统UI的设计语言
* 自动获取TMDB元数据信息
* 支持AniList跟踪
* 支持代理服务器协议（HTTP、SOCKS4、SOCKS5）
# 白嫖 
[Miru github地址](https://github.com/miru-project/miru-app "Miru github地址")
[Miru 官网](https://miru.js.org/ "Miru 官网")
## Gopeed (下载器)
![Gopeed](https://github.com/GopeedLab/gopeed/blob/main/_docs/img/banner.png "gopeed logo")
一个现代化的下载管理器，支持所有平台。由Golang和Flutter构建
# 介绍
* Gopeed（全称Go Speed），由Golang + Flutter开发的高速下载器，支持（HTTP，BitTorrent，Magnet）协议，支持所有平台。除了基本的下载功能，Gopeed还是一个高度可定制的下载器，支持通过与API集成或安装和开发扩展来实现更多功能。
# 白嫖 
[Gopeed github地址](https://github.com/GopeedLab/gopeed "Gopeed github地址")
[Gopeed 官网](https://gopeed.com/ "Gopeed 官网")
## Mihon （漫画）
![mihon](https://github.com/mihonapp/mihon/raw/main/.github/assets/logo.png "mihon logo")  
* Tachiyomi之后的再创项目
* （Tachiyomi 项目开发者收到来自韩国 Kakao 娱乐公司的法律威胁，要求销毁该项目并确保删除所有分叉。 1月9日，Tachiyomi 发布公告表示：由于特殊情况，决定清理扩展列表以确保 Tachiyomi 的长期可持续性。 随后开发者发布公告表示不再维护软件并逐步关闭官方开发社区。）
# 介绍 
* 一个漫画阅读器
# 扩展库导入 
[Keiyoushi](https://keiyoushi.github.io/ "扩展库Keiyoushi")（软件不自带插件库，需自行导入）
* 方法一：
打开扩展网站，下载想要使用的图源到电脑上
* 方法二：
进入设置 - Browse（浏览） - Extension repositories（扩展仓库） - Add repository（添加仓库）
[扩展库Keiyoushi订阅地址](https://keiyoushi.github.io/add-repo?url=https%3A%2F%2Fraw.githubusercontent.com%2Fkeiyoushi%2Fextensions%2Frepo%2Findex.min.json "扩展库Keiyoushi订阅地址")
# 白嫖 
[mihon github地址](https://github.com/mihonapp/mihon "mihon github地址")
[mihon 官网](https://mihon.app/ "mihon 官网")
## MusicFree （音乐）
* 不多说上链接[MusicFree 官网](https://musicfree.upup.fun/ "MusicFree 官网")
## Easy Cat Timer (简单计时器)windows
![Easy Cat Timer](https://github.com/xujiangjiang/Easy-Cat-Timer/raw/master/ReadMeImage/MainWindow.png "Easy Cat Timer")
# 白嫖
* 2019年7月2日之后，你可以在Steam商店里，下载这款软件。
最新版本： v1.0.1.1 (2019年09月13日)
下载地址
Steam地址：<https：//store.steampowered.com/app/1054580/Easy_Cat_Timer/>
* [Easy Cat Timer github地址](https://github.com/xujiangjiang/Easy-Cat-Timern "Easy Cat Timer github地址")
[Easy Cat Timer 官网](https://easycattimer.com/ "Easy Cat Timer 官网")
## Legado / 开源阅读
 ![ledado](https://github.com/gedoor/legado/raw/master/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png "legado logo")
# 白嫖 
[Legado github地址](https://github.com/gedoor/legado "Legado github地址")
[Legado](https://www.legado.top/ "Legado 官网")
## 椒盐音乐 Salt Player 
# 介绍
* 椒盐音乐是一款 Android 平台本地音乐播放软件。
# 白嫖 
[Salt Player github地址](https://github.com/Moriafly/SaltPlayerSourceo "Salt Player github地址")
[Salt Player 官网](https://moriafly.xyz/SaltPlayerSource/index.html#/ "Salt Player 官网")
## 就到这里

**Comment by Harmony-Mushang** [comment]
为什么不能点赞？

**Reply by 6078HITHEDAY** [reply]
在GitHub里

**Comment by HKLHaoBin** [comment]
这是把整个 Markdown 复制下来了呀

**Reply by 6078HITHEDAY** [reply]
我自己的博客
https://6078hitheday.github.io

**Comment by rong-xiaoli** [comment]
盲猜自己的博客复制了一篇文章（

**Reply by 6078HITHEDAY** [reply]
嘿嘿
